spec/sms_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'sms'
+
+describe SMS do
+
+  subject(:sms) { described_class.new }
+
+  it 'send a text message' do
+   expect(sms).to respond_to(:send_sms)
+ end
```

Comment:
Lint/BlockAlignment: `end` at 9, 1 is not aligned with `it 'send a text message' do` at 7, 2.